Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -199,6 +234,36 @@ click: function () { closeClicked.raiseEvent(this); }');
 
         return destroyObject(this);
     };
+    /**
+     * Receives a string and enters it to the DOM
+     * If the string contains a <script> tag, it is
+     * parsed using contextualFragment
+     *
+     * @type {Function}.
+     * @private
+     */
+    /**
+     *
+     * @param s {String}
+     * @param frame {Element}
+     * @param allowScripts {Boolean}
+     */
+    InfoBox.stringToHtml = function stringToHtml(s, frame, allowScripts) {
+        // if we don't have a script tag - in case the method was used as static...
+        if (allowScripts && s.indexOf('<script') > -1 && s.indexOf('</script>') > -1) {
+            // we have a script tag - let's handle it
+            // check for modern browsers
+            if (defined(document.createRange) && defined(document.createRange().createContextualFragment)) {
```

Comment:
Do we know of a case where a WebGL supporting browser does not also support `createContextualFragment`?  I'm all for defensive coding, but I think every browser Cesium officially supports has this functionality.